test/specs/markup/sample.yaml

```diff
@@ -0,0 +1,17 @@
+#
+# markup configuration for theblog
+#
+version: 1
+
+markup:
+  images-in-gallery:
+    match: section.is-gallery image
+    wrap: <div class="gallery-image">
```

Comment:
I'm worried this could become an XSS attack vector if we accept HTML here…
I'd love to see us use the same CSS selector syntax instead, like:
```
wrap: div.gallery-image // => <div class="gallery-image">…</div>
wrap: section.foo bar .baz // => <section class="foo"><bar><div class="baz">…</div></bar></section>
wrap: foo[bar="baz"] // => <foo bar="baz">…</foo>
…
```

Maybe by using somthing like [emmet](https://github.com/emmetio/emmet) module.

Also has the side benefit of using 1 consistent notation in all properties

Reply:
I like the emmet idea. Do you want to update the schema and tests to unify this to one single property?

Reply:
Not sure what you mean by "single property", the rest is quite easy as it only requires string replacements. we do not actually test for any HTML value at the moment

Reply:
Instead of wrap/class/attribute, just have emmet to handle all cases.

Reply:
I don't think that's going to be possible with emmet. It does not modify pre-existing markup, only generates new ones from the shorthand notation.

So assuming we have a pre-existing `<img src="…"/>`, I don't see how to specify a class/attribute equivalent and achieve something like `<img class="gallery-image" src="…" data-foo="bar"/>`.

Reply:
Ok.